### DISCUSSION TITLE
Services are not running after updating SOUP

### DISCUSSION THREAD

**Discussion by IzacFrank** [Security-Onion-Solutions/securityonion#11655] [General] [answered]
Hello everyone,

I was updating from 2.3.260 to 2.3.270. I had issues during the SOUP update and then reverted the snapshot and I managed to update SOUP then during the second SOUP (OS and application patches), it complained that yumlock was enabled and I rebooted the VM and services didn't stop (starting and waiting-start).

I tried to run them one by one and only a few of them responded. Please advise. 
I see the below error mostly.
sudo salt-call state.highstate
[ERROR   ] Unable to connect pusher: Stream is closed

This is a standalone server

Thank you.
Isac

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/93327361/59d0e173-9d7d-4047-92a3-17eb568b5e43)

**Comment by IzacFrank** [comment]
some services say "No such container: so-curator" when I try sudo so-curator-restart.
Can I somehow reinstall it? 
How can I back up my config, reinstall from scratch, and import the configuration?



Error response from daemon: No such container: so-redis
Error: No such container: so-redis


Thank you.

**Comment by cm-ops** [comment]
Try to run `so-checkin` and see if there are errors. Also, look in the minion log - `/opt/so/log/salt/minion`.

**Reply by IzacFrank** [reply]
@cm-ops thank you for your reply.  I ran so-checking and it stuck at "[INFO    ] Executing command 'yum' in directory '/root'"
I had been waiting there for more than an hour. Before that I managed to run all services except "SO-SOC"

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/93327361/5e7cf51a-58a0-42f7-ba67-4d6ae0ce5295)

**Reply by IzacFrank** [reply, marked as answer]
I started most of the services one by one like "so-elasticsearch-restart" like the first picture but a few more. Then run "sudo salt-call state.highstate", and all other services started except so-soc.
I executed SOUP and it solved all the problems, all services running now.
